### PR TITLE
ADEN-7334 Adds getPublisherConsent command

### PR DESCRIPTION
**Pull request by leshniak**

### CODE REVIEW

File: extensions/wikia/TrackingOptIn/js/cmp.js [hunk 1]
```diff
@@ -227,6 +266,25 @@ define('wikia.cmp', [
 				}, {})
 			}, true);
 		},
+		getPublisherConsent: function (purposesIds, callback) {
+			var purposesAllowed = (purposesIds || publisherConsent.getPurposesAllowed()).reduce(function (obj, id) {
+					var type = (id > 24) ? 'customPurposeConsents' : 'standardPurposeConsents';
```
Comment (nandy-andy) [inline extensions/wikia/TrackingOptIn/js/cmp.js]: What's the magic `24`? Can we assign it to a variable with a descriptive name? ;-)

File: extensions/wikia/TrackingOptIn/js/cmp.js [hunk 2]
```diff
@@ -47,9 +60,31 @@ define('wikia.cmp', [
 		log('Initializing module', log.levels.debug, logGroup);
 
 		vendorConsentString = cookies.get(vendorConsentCookieName);
+		publisherConsentString = cookies.get(publisherConsentCookieName);
+
+		if (publisherConsentString) {
```
Comment (nandy-andy) [inline extensions/wikia/TrackingOptIn/js/cmp.js]: This `if-else` logic is very similar as for publisher consent string and vendor consent string - can we extract it and re-use it in the `init`?

File: extensions/wikia/TrackingOptIn/js/cmp.js (hunk 1 above)
Comment (ludwikkazmierczak) [reply extensions/wikia/TrackingOptIn/js/cmp.js]: @nandy-andy id's up to 24 are reserved. 25+ may be our custom purposes :)

File: extensions/wikia/TrackingOptIn/js/cmp.js (hunk 2 above)
Comment (leshniak) [reply extensions/wikia/TrackingOptIn/js/cmp.js]: yes, it's similar. we could, but the next step is to remove all this code. I think that it's not worth of time.

**Review by dmnsobczak — APPROVED** [review]

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**